crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -505,6 +507,10 @@ pub struct PickledAccount {
     pub shared: bool,
     /// The number of uploaded one-time keys we have on the server.
     pub uploaded_signed_key_count: u64,
+    /// The local time creation of this account (seconds since epoch), used as
+    /// creation time of own device
+    #[serde(default)]
+    pub creation_local_time_ts: UInt,
```

Comment:
Why is this one a plain UInt?

Reply:
Also `time` + `ts` is a bit redundant, no?

Reply:
updated to creation_local_time

Reply:
Right, but the original question still stands, why is it a `UInt` here when it's `MilliSecondsSinceUnixEpoch` elsewhere?

Reply:
missed that uses MilliSecondsSinceUnixEpoch now